lib/client/player.go

```diff
@@ -133,7 +134,7 @@ func timestampFrame(message string) {
 	defer esc(restoreCursor)
 
 	// move cursor to -10:0
-	esc(fmt.Sprintf("[%d;%df", 0, int(sz.Width)-len(message)))
+	esc(fmt.Sprintf("[%d;%df", 0, int(width)-len(message)))
```

Comment:
It would be off and result in some graphical strangeness. That said, the same problem also exists with any ANSI escapes in the message, so we'd want to filter those out to be totally correct here.

That said, we control the message here, and it is only ever set to a timestamp. Perhaps fixing this bug is out of scope for this PR?

Reply:
Ah yes I guess we're okay if the message is always a timestamp that we control.

Reply:
I've added a TODO note about the issue to be safe